include/eradius_lib.hrl

```diff
@@ -92,7 +98,13 @@
           accessRejects            = 0 :: non_neg_integer(),
           accessChallenges         = 0 :: non_neg_integer(),
           accountRequests          = 0 :: non_neg_integer(),
+          accountRequestsStart     = 0 :: non_neg_integer(),
```

Comment:
Nothing wrong, just wondering if there is a way to define the same record members once, instead of multiple times.

Reply:
@ebengt yes, probably it would be good idea to merge records into one, but in separate PR as this one will bring more unrelated changes